### PR TITLE
First implementation of JDBC without many to many.

### PR DESCRIPTION
**Pull request by vminin-solvd**
Please check if my JDBC implementation is correct minus the OrderHasAdditionalService many to many relationship.

### CODE REVIEW

File: src/main/java/com/solvd/interfaces/iEmployeeDAO.java [hunk 1]
```diff
@@ -1,7 +1,8 @@
 package com.solvd.interfaces;
 
 import com.solvd.models.Employee;
+import java.util.List;
 
 public interface iEmployeeDAO extends IBaseDAO<Employee> {
```
Comment (ankochergin) [inline src/main/java/com/solvd/interfaces/iEmployeeDAO.java]: Remember about naming conventions, fix everywhere

File: src/main/java/com/solvd/interfaces/iEmployeeDAO.java [hunk 2]
```diff
@@ -1,7 +1,8 @@
...
-Employee getEmployeeByLastName(String lastName);
+List<Employee> getEmployeeByLastName(String lastName);
 }
```
Comment (ankochergin) [inline src/main/java/com/solvd/interfaces/iEmployeeDAO.java]: reformat all files in your project, formatting is broken now

File: src/main/java/com/solvd/jbdc/dao/AdditionalServiceDAO.java [hunk 3]
```diff
@@ -45,18 +49,19 @@ public List<AdditionalService> getAll(){
                 while (rs.next()) {
                     AdditionalService additionalService = new AdditionalService();
                     additionalService.setId(rs.getInt("id"));
-                    additionalService.setServiceType(rs.getString("service_type"));// FIXME
+                    ServiceTypeDAO serviceTypeDAO = new ServiceTypeDAO();
+                    additionalService.setServiceType(serviceTypeDAO.getServiceTypeByName(rs.getString("service_type")));
```
Comment (ankochergin) [inline src/main/java/com/solvd/jbdc/dao/AdditionalServiceDAO.java]:
```suggestion
                    additionalService.setServiceType(new ServiceTypeDAO().getServiceTypeByName(rs.getString("service_type")));
```

Comment (ankochergin) [reply src/main/java/com/solvd/jbdc/dao/AdditionalServiceDAO.java]: update same things in all methods in all DAO classes

File: src/main/java/com/solvd/jbdc/dao/CustomerDAO.java [hunk 4]
```diff
@@ -45,19 +50,19 @@ public List<Customer> getAll(){
                 while (rs.next()) {
                     Customer Customer = new Customer();
                     Customer.setId(rs.getInt("id"));
-                    Customer.setFirstName(rs.getString("first_name"));// FIXME
-                    Customer.setLastName(rs.getString("last_name"));// FIXME
+                    Customer.setFirstName(rs.getString("first_name"));
+                    Customer.setLastName(rs.getString("last_name"));
```
Comment (ankochergin) [inline src/main/java/com/solvd/jbdc/dao/CustomerDAO.java]: naming conventions

File: src/main/java/com/solvd/jbdc/dao/EmployeeDAO.java [hunk 5]
```diff
@@ -46,19 +51,22 @@ public List<Employee> getAll(){
...
                     Employee employee = new Employee();
                     employee.setId(rs.getInt("id"));
-                    employee.setFirstName(rs.getString("first_name"));// FIXME
-                    employee.setLastName(rs.getString("last_name"));// FIXME
+                    employee.setFirstName(rs.getString("first_name"));
+                    employee.setLastName(rs.getString("last_name"));
+                    Position position = new Position();
+                    position.setId(rs.getInt("position_id"));
+                    employee.setPosition(position);
                     Employees.add(employee);
```
Comment (ankochergin) [inline src/main/java/com/solvd/jbdc/dao/EmployeeDAO.java]: naming conventions

File: src/main/java/com/solvd/jbdc/dao/ManufacturerDAO.java [hunk 6]
```diff
@@ -37,29 +42,29 @@ public void saveEntity(Manufacturer manufacturer) {
...
             ps.execute();
             try (ResultSet rs = ps.getResultSet()) {
                 while (rs.next()) {
-                    Manufacturer Manufacturer = new Manufacturer();
-                    Manufacturer.setId(rs.getInt("id"));
-                    Manufacturer.setManufacturerName(rs.getString("manufacturer_name"));// FIXME
-                    Manufacturers.add(Manufacturer);
+                    Manufacturer manufacturer = new Manufacturer();
+                    manufacturer.setId(rs.getInt("id"));
+                    manufacturer.setManufacturerName(rs.getString("manufacturer_name"));// FIXME
```
Comment (ankochergin) [inline src/main/java/com/solvd/jbdc/dao/ManufacturerDAO.java]: remove comments if it's fixed

File: src/main/java/com/solvd/models/OrderHasAdditionalService.java [hunk 7]
```diff
@@ -0,0 +1,40 @@
+package com.solvd.models;
+
+public class OrderHasAdditionalService {
+    private int id;
+    private Order order;
+    private AdditionalService additionalService;
```
Comment (ankochergin) [inline src/main/java/com/solvd/models/OrderHasAdditionalService.java]: In the conjunction table we don't have id

File: src/main/java/com/solvd/interfaces/iEmployeeDAO.java (hunk 1 above)
Comment (vminin-solvd) [reply src/main/java/com/solvd/interfaces/iEmployeeDAO.java]: This is in reference to the interface naming convention correct? I fixed these errors.

File: src/main/java/com/solvd/interfaces/iEmployeeDAO.java (hunk 2 above)
Comment (vminin-solvd) [reply src/main/java/com/solvd/interfaces/iEmployeeDAO.java]: Fixed with auto format in github push settings in IDEA

File: src/main/java/com/solvd/jbdc/dao/CustomerDAO.java (hunk 4 above)
Comment (vminin-solvd) [reply src/main/java/com/solvd/jbdc/dao/CustomerDAO.java]: Fixed

File: src/main/java/com/solvd/jbdc/dao/EmployeeDAO.java (hunk 5 above)
Comment (vminin-solvd) [reply src/main/java/com/solvd/jbdc/dao/EmployeeDAO.java]: Fixed

File: src/main/java/com/solvd/jbdc/dao/ManufacturerDAO.java (hunk 6 above)
Comment (vminin-solvd) [reply src/main/java/com/solvd/jbdc/dao/ManufacturerDAO.java]: Fixed the FIXME

File: src/main/java/com/solvd/models/OrderHasAdditionalService.java (hunk 7 above)
Comment (vminin-solvd) [reply src/main/java/com/solvd/models/OrderHasAdditionalService.java]: Fixed

File: src/main/java/com/solvd/jbdc/dao/AdditionalServiceDAO.java (hunk 3 above)
Comment (vminin-solvd) [reply src/main/java/com/solvd/jbdc/dao/AdditionalServiceDAO.java]: Fixed

File: .idea/uiDesigner.xml
Comment (ankochergin) [inline .idea/uiDesigner.xml:1]: please remove .idea folder from git

File: src/main/java/com/solvd/jbdc/dao/AdditionalServiceDAO.java [hunk 8]
```diff
@@ -11,7 +13,8 @@
 import java.util.ArrayList;
 import java.util.List;
 
-public class AdditionalServiceDAO implements iAdditionalServiceDAO {
+public class AdditionalServiceDAO implements IAdditionalServiceDAO {
+    private static final Logger LOGGER = LogManager.getLogger(AdditionalServiceDAO.class);
```
Comment (ankochergin) [inline src/main/java/com/solvd/jbdc/dao/AdditionalServiceDAO.java]:
```suggestion
public class AdditionalServiceDAO implements IAdditionalServiceDAO {

    private static final Logger LOGGER = LogManager.getLogger(AdditionalServiceDAO.class);
```

Comment (ankochergin) [inline src/main/java/com/solvd/jbdc/dao/AdditionalServiceDAO.java]: It's not necessary to make logger `static`

**Review by ankochergin — COMMENTED** [review]
+ please reformat code

**Pull request merged**